app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -862,22 +886,22 @@ export default {
     clearEditorSelection() {
       this.updateEditorSelectionWith = '';
     },
-    insertEmoji(emoji, selectionStart, selectionEnd) {
+    insertIntoTextEditor(emoji, selectionStart, selectionEnd) {
       const { message } = this;
       const newMessage =
         message.slice(0, selectionStart) +
         emoji +
         message.slice(selectionEnd, message.length);
       this.message = newMessage;
     },
-    emojiOnClick(emoji) {
+    addIntoEditor(emoji) {
       if (this.showRichContentEditor) {
         this.updateEditorSelectionWith = emoji;
         this.onFocus();
       }
       if (!this.showRichContentEditor) {
         const { selectionStart, selectionEnd } = this.$refs.messageInput.$el;
-        this.insertEmoji(emoji, selectionStart, selectionEnd);
+        this.insertIntoTextEditor(emoji, selectionStart, selectionEnd);
```

Comment:
This code is very hard to understand. It expects emoji in the function. However you are passing a string in the above function. Please re-consider the implementation.

Reply:
Makes sense, I have renamed the variables so it will be generic `text` everywhere.